crates/ibc/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -253,6 +254,83 @@ impl ClientState {
             _ => Ok(()),
         }
     }
+
+    fn check_trusted_header(
+        consensus_state: &TmConsensusState,
+        header: &Header,
```

Comment:
Since this is not a full header check and also the header is in fact untrusted and the consensus state is trusted, maybe:

```suggestion
    fn check_header_validator_set(
        trusted_consensus_state: &TmConsensusState,
        header: &Header,
```

Reply:
https://github.com/cosmos/ibc-rs/pull/215/commits/99ccdf5112465879b4e2b104f1a8556bdcfb0407